### PR TITLE
Update tarball testing instructions

### PR DESCRIPTION
**Pull request by timtheisen**
I tested out the new script to paste and ensured that it works

### CODE REVIEW

File: docs/release/cut-sw-release.md [hunk 1]
```diff
@@ -168,34 +182,30 @@ NON_UPCOMING_VERSIONS=<NON-UPCOMING VERSION(S)>
...
+        rm -rf $rhel-$arch
+    fi
 }
 
 pushd /tmp
 
-for client in osg-afs-client osg-wn-client; do
-    for ver in $NON_UPCOMING_VERSIONS; do
+for ver in $NON_UPCOMING_VERSIONS; do
+    for client in osg-afs-client osg-wn-client; do
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: What's the reason for switching up the for loops?

File: docs/release/cut-sw-release.md [hunk 2]
```diff
@@ -157,7 +157,21 @@ cd release-tools
...
+        - osg-wn-client, el6: 23 megabytes
+    -   64-bit
+        - osg-wn-client, el6: 23 megabytes
+        - osg-wn-client, el7: 31 megabytes
+-   OSG 3.4.x
+    -   64-bit
+        - osg-afs-client, el6: 23 megabytes
+        - osg-afs-client, el7: 31 megabytes
+        - osg-wn-client, el6: 23 megabytes
+        - osg-wn-client, el7: 31 megabytes
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: Do we need to be so specific about the tarball sizes? It's also not clear what the user of the doc should be doing with this information. If there's a size delta, is the release broken? How big of a size delta is worth bringing to the attention of the release manager? Can't this whole section be checked programmatically?

File: docs/release/cut-sw-release.md [hunk 3]
```diff
@@ -168,34 +182,30 @@ NON_UPCOMING_VERSIONS=<NON-UPCOMING VERSION(S)>
...
 dotest () {
     file=$dir/$client-$ver-1.$rhel.$arch.tar.gz
-    mkdir -p $rhel-$arch
-    pushd $rhel-$arch
-    tar xzf ../$file
-    $client/osg/osg-post-install
-    $client/osgrun osg-version
-    popd
-    rm -rf $rhel-$arch
+    if [ -e $file ]; then
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: I'm not a big fan of the existence check. Presently, we expect 7 tarballs and if one of them doesn't exist, the tests should complain. If more than 7 tarballs are created, the tests should probably fail, too.

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: I reworked the script to complain of missing tarballs. It still does not report extra tarballs. In any case, this is an improvement.

File: docs/release/cut-sw-release.md (hunk 1 above)
Comment (timtheisen) [reply docs/release/cut-sw-release.md]: You don't know which client to test if you don't know the major version.

File: docs/release/cut-sw-release.md (hunk 2 above)
Comment (timtheisen) [reply docs/release/cut-sw-release.md]: I'll rework the test script to check the size programatically. Then we can drop this change.

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: I pushed the reworked script and I tested functionality of the script.

**Comment by timtheisen** [comment]
Hey Brian,

Can you review this again?

...Tim

File: docs/release/cut-sw-release.md [hunk 4]
```diff
@@ -157,7 +157,8 @@ cd release-tools
 ./1-client-tarballs $NON_UPCOMING_VERSIONS
 ```
 
-You should get up to 8 tarballs for each version (excluding upcoming), 25-55 megs each and they should all have the version number in the name.
+You should get 7 tarballs.
+They should all have the version number in the name.
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: These sentences seem superfluous if we're automatically testing the existence of the 7 tarballs below.

**Comment by timtheisen** [comment]
I have tested the script with missing tarballs and I also tested the too big code by temporarily setting a smaller acceptable size.

**Pull request merged**